test/sql/pattern_matching/inheritance_support.test

```diff
@@ -65,6 +65,25 @@ FROM GRAPH_TABLE(pg
 3	Peter	0	VU
 4	David	3	CWI
 
+query IIII
```

Comment:
Before commit https://github.com/cwida/duckpgq-extension/pull/193/commits/aadd10cad5ba0d7f5da19274a601508b196cafe1, query result is
```
id      name    id_1    mask    name_1  personId        organisationId
BIGINT  VARCHAR BIGINT  BIGINT  VARCHAR BIGINT  BIGINT
[ Rows: 8]
0       Daniel  1       1       UvA     0       1
0       Daniel  2       2       EY      0       2
0       Daniel  3       2       CWI     0       3
1       Tavneet 2       2       EY      1       2
1       Tavneet 3       2       CWI     1       3
2       Gabor   3       2       CWI     2       3
3       Peter   0       1       VU      3       0
4       David   3       2       CWI     4       3
```